scripts/lint-re-frame-in-quo-components.sh

```diff
@@ -1,11 +1,9 @@
 #!/usr/bin/env sh
 
-CHANGES=$(git diff --no-ext-diff --diff-filter=d --cached --unified=0 --no-prefix --minimal --exit-code src/quo src/quo2 | grep '^+' || echo '')
-
-INVALID_CHANGES=$(echo "$CHANGES" | grep -E '(re-frame/dispatch|rf/dispatch|re-frame/subscribe|rf/subscribe|rf/sub|<sub|>evt|status-im)')
+INVALID_CHANGES=$(grep -E -r '(re-frame/dispatch|rf/dispatch|re-frame/subscribe|rf/subscribe|rf/sub|<sub|>evt|status-im\.|status-im2\.|utils\.i18n)' './src/quo2')
```

Comment:
@flexsurfer, I agree with all the changes here, except for one.

What exactly do we gain by using `quo2/i18n` versus using `utils.i18n`? What problem do you see that's been caused by using `utils.i18n`? Now there's also an abstraction to configure the `label-fn-atom`, but why do we need this extra complexity?

Reply:
yes, I thought we agreed to share the utils for both quo2 and status_2? 🤔 or is there something I am missing about this?

Reply:
it's possible to use quo2 outside status, actually, that's the idea of having a separate namespace (library) otherwise why do we need a separate namespace, with the magic name QUO library :) probably someday it will be moved in a separate repo
it's hard to explain,  at the moment we don't gain anything, but if we decided to develop it as a separate library albeit in the same repository lets try to keep it as separate library without status-im deps, yeah i18n is utility which is not related to the app, but resources injected, mhm, ok seems like you're right, if resources won't be injected in i18n utils, it will work the same, so no difference :) makes sense, i'll rollback this change

Reply:
ok cool, as I understand this still retains quo2 as a separate library but we can see utils as it's own package which quo2 and status_im2 both share. Either way seems fine to me, just as long as we're aware of the decisions made 👍

Reply:
>it's possible to use quo2 outside status, actually, that's the idea of having a separate namespace (library) otherwise why do we need a separate namespace, with the magic name QUO library :) probably someday it will be moved in a separate repo

I've heard many times in the team that we don't want to create a true component library, nor make it external to the status-mobile repository. The abstraction you created is leaning in that direction, but I don't think we need any of this. If it's hard to answer my questions it's a good indication that we don't need this extra complexity.

I'm just bringing the YAGNI argument. If we start to go in this route we'll need to revisit many other assumptions that are tightly coupled with the rest of the status codebase. But why? What real value do they bring? Perhaps in the distant future we might want to treat `quo2` as a true separate library, but now, IMO, let's just keep things simple.

>that's the idea of having a separate namespace (library) otherwise why do we need a separate namespace

The point of having a separate directory for `quo2` is like all the other directories, to create a clean separation of concerns. It doesn't mean `quo2` is being designed for being a separate library.

But @flexsurfer, what you did is indeed interesting, I just don't see it as solving a problem that we have, nor one that we'll have in medium even long term.